### PR TITLE
C20 wmde test 01

### PR DESCRIPTION
**Pull request by Abban**
https://phabricator.wikimedia.org/T256611
Layout test ticket: https://phabricator.wikimedia.org/T256621

### CODE REVIEW

**Comment by Abban** [comment]
Works okay in IE11

File: desktop_preact/components/BannerText.jsx [hunk 1]
```diff
@@ -3,39 +3,62 @@ import { h } from 'preact';
...
-				className="optional-text text-m">, um Wikipedias Unabhängigkeit zu sichern</span>:
-		</p>
-		<p className="text__paragraph">
-			<span>{ campaignDaySentence } Wikipedia wird durch Spenden von durchschnittlich 23,83 € finanziert, aber 99 % der Leserinnen und Leser
-				spenden nicht. </span>
+		<div className="banner-text-inner">
+			<div className="banner-text-ctrl">
+				<p>
+					Liebe Leserinnen und Leser, bitte verzeihen Sie die Störung. Es ist ein bisschen unangenehm, daher
+					kommen wir gleich zur Sache. An diesem { capitalizeFirstLetter( weekdayPrepPhrase ) }  { currentDayName } sind Sie in Deutschland gefragt:
```
Comment (gbirke) [inline desktop_preact/components/BannerText.jsx]:
```suggestion
					kommen wir gleich zur Sache. { capitalizeFirstLetter( weekdayPrepPhrase ) }  { currentDayName } sind Sie in Deutschland gefragt:
```

File: desktop_preact/components/BannerText.jsx [hunk 2]
```diff
@@ -3,39 +3,62 @@ import { h } from 'preact';
...
+				</p>
+				<p>
+					{ campaignDaySentence }
+					<strong>Wikipedia wird durch Spenden von durchschnittlich 22,81 € finanziert. Wenn alle, die das jetzt
+						lesen, einen kleinen Beitrag leisten, wäre unser Spendenziel bereits am heutigen { currentDayName }
+						erreicht. Millionen Menschen nutzen Wikipedia, aber 99 % der Leserinnen und Leser spenden nicht. </strong>
```
Comment (gbirke) [inline desktop_preact/components/BannerText.jsx:18]: Bildschirmfoto 2020-07-10 um 17 56 39

File: desktop_preact/components/BannerText.jsx [hunk 3]
```diff
@@ -3,39 +3,62 @@ import { h } from 'preact';
...
+					<li>
+						<strong>Wikipedia wird durch Spenden von durchschnittlich 22,81 € finanziert.</strong>
+					</li>
+					<li>
+						<strong>Wenn alle, die das jetzt lesen, einen kleinen Beitrag leisten, wäre unser Spendenziel bereits am
+							heutigen { currentDayName } erreicht.</strong>
+					</li>
+					<li>
+						<strong>Millionen Menschen nutzen Wikipedia, aber 99 % der Leserinnen und Leser spenden nicht.</strong>
+					</li>
```
Comment (gbirke) [inline desktop_preact/components/BannerText.jsx:47]: The list bullets show up as black in firefox, I fear you have to add [custom styling](https://www.w3.org/Style/Examples/007/color-bullets.en.html)

**Review by gbirke — CHANGES_REQUESTED** [review]
Please add `<span>` tags around placeholders to avoid whitespace problems, as discussed.

The 2 commits are not like I would like them to be: The purpose of having 2 commits is supporting the reviewer and your future self to answer the question: "What was actually different between CTRL and VAR"? In this case it's a bit silly, because the 1st commit would have the same text twice in `BannerText.jsx`. The things that would be changed in the 2nd commit would be `bannerType` in `banner_var.js`,  `BannerText.jsx` and the CSS snippet in `BannerText.pcss`. For cases like these, where you introduce a new feature in both banners, I like to do 3 commits: 1st for setting up the campaign with the "winner" of the last test, 2nd for adding the feature (in this case adding the prop, enum and class name), the 3rd for introducing the difference.

This explanation is meant more like a clarification and guideline and I won't force you to interactively rebase and pry everything apart. The changes requested are for the code, not the commits.

**Pull request merged**